core/src/main/scala/kafka/coordinator/transaction/TransactionStateManager.scala

```diff
@@ -47,7 +47,7 @@ object TransactionStateManager {
   // default transaction management config values
   val DefaultTransactionsMaxTimeoutMs: Int = TimeUnit.MINUTES.toMillis(15).toInt
   val DefaultTransactionalIdExpirationMs: Int = TimeUnit.DAYS.toMillis(7).toInt
-  val DefaultAbortTimedOutTransactionsIntervalMs: Int = TimeUnit.MINUTES.toMillis(1).toInt
+  val DefaultAbortTimedOutTransactionsIntervalMs: Int = TimeUnit.SECONDS.toMillis(10).toInt
```

Comment:
Is this part of the KIP to change the default value?

Reply:
Do we still want this change, even if Kafka Streams changes slip? Maybe ok, because people might write plain consumer/producer apps using the new fencing. Just double checking.

Reply:
The KIP and mailing list just get updated with this minor change FYI